chart/values.yaml

```diff
@@ -196,6 +196,18 @@ mariadb:
     persistence:
       size: 1G
 
+elasticsearch:
+  enabled: true
+  #Choose versions in https://www.docker.elastic.co/ from "Elasticsearch" list
+  version: 6.5.0
+  persistence:
+    size: 512Mi
+    storageClassName: nfs
+  requests:
+    cpu: 25m
+    memory: 512Mi
+    heapSize: 256m
```

Comment:
The heapSize is a parameter for the resources _within_ the container, it should not be mixed with resource configuration for the container itself.